src/components/pages/Home.tsx

```diff
@@ -77,6 +92,26 @@ function SectionQueryResult({
 }
 
 export default function HomePage(): ReactElement {
+  const [searchDids, setSearchDIDs] = useState<string>()
+  const { config } = useOcean()
+
+  useEffect(() => {
+    getHighestLiquidityDIDs().then((results) => {
+      setSearchDIDs(results)
+    })
+  }, [config?.metadataCacheUri, searchDids])
```

Comment:
since we query the subgraph and not aquarius in this effect we should switch dependency to `config?.subgraphUri`